yt/yt/server/queue_agent/queue_exporter.cpp

```diff
@@ -315,7 +344,7 @@ class TQueueExportTask
             transactionId);
 
         TaskInstant_ = TInstant::Now();
-        ExportUnixTsUpperBound_ = GetExportUnixTsUpperBound(TaskInstant_, ExportConfig_->ExportPeriod);
+        ExportUnixTsUpperBound_ = GetExportUnixTsUpperBound(TaskInstant_, ExportConfig_->ExportSchedule);
```

Comment:
If done via config inheritance, this will just be an upcast. Or you can keep the fields in the config and make a helper struct which is retrieved via ExportConfig_->GetExportSchedule(). 

You could also store this helper-struct right in the config, but I don't see a good reason to, constructing a struct is free.